config.yaml

```diff
@@ -31,9 +31,9 @@ categories:
     tags: [test]
 
 catalogs:
-  - name: tektoncd
+  - name: catalog
+    org: tektoncd
     type: community
-    owner: tektoncd
     url: https://github.com/tektoncd/catalog
     revision: master
     contextDir: /
```

Comment:
this is supposed to be optional and treated as "" by default.

Reply:
Yeah. I think we should remove this field from here and at the bottom of config file mention all fields their explanation

Reply:
+1